### PR TITLE
Fix compressor op_types

### PR DESCRIPTION
**Pull request by chicm-ms**

### CODE REVIEW

File: examples/model_compress/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ configure_list = [{
     'start_epoch': 0,
     'end_epoch': 10,
     'frequency': 1,
-    'op_type': 'default'
+    'op_types': 'default'
```
Comment (QuanluZhang) [inline examples/model_compress/README.md]: missed square brackets?

Comment (chicm-ms) [reply examples/model_compress/README.md]: Thanks, updated.

File: src/sdk/pynni/nni/compression/torch/compressor.py [hunk 2]
```diff
@@ -35,7 +35,7 @@ def compress(self, model):
         for name, module in model.named_modules():
             layer = LayerInfo(name, module)
             config = self._select_config(layer)
-            if config is not None:
+            if config is not None and layer.type in config['op_types']:
```
Comment (QuanluZhang) [inline src/sdk/pynni/nni/compression/torch/compressor.py]: _select_config already checked whether `layer.type in config['op_types']`?

File: src/sdk/pynni/nni/compression/torch/compressor.py [hunk 3]
```diff
@@ -112,10 +121,6 @@ class Quantizer(Compressor):
     Base quantizer for pytorch quantizer
     """
 
-    def __call__(self, model):
```
Comment (QuanluZhang) [inline src/sdk/pynni/nni/compression/torch/compressor.py]: why removing this func?

File: src/sdk/pynni/nni/compression/torch/compressor.py (hunk 2 above)
Comment (chicm-ms) [reply src/sdk/pynni/nni/compression/torch/compressor.py]: Good catch, removed.

File: src/sdk/pynni/nni/compression/torch/compressor.py (hunk 3 above)
Comment (chicm-ms) [reply src/sdk/pynni/nni/compression/torch/compressor.py]: There is an exactly same version already in parent class Compressor.

Comment (QuanluZhang) [reply src/sdk/pynni/nni/compression/torch/compressor.py]: i see

**Pull request merged**